Chapter5/stats_handler.erl

```diff
@@ -0,0 +1,13 @@
+-module(stats_handler).
```

Comment:
That's fine, in a first version, I think, though, it seems to me that `Db` is an internal variable, that client shouldn't even know about.